### PR TITLE
WIP: Let's chatops JIRA so you don't have to deal with their UI

### PR DESCRIPTION
**Pull request by lakshmi-kannan**
### Status

Replace this: describe the pack and PR status. Examples: 
- new pack, complete
- pack extension, work in progress
- bugfix
### Description

Replace this with the PR description.
### Checklist (tick everything that applies)
- [ ] Metadata: pack.yaml, icon, structure (required for new packs)
- [ ] Version bump (required for changed packs)
- [ ] Code linting (required, can be done after the PR checks)
- [ ] [Tests](https://docs.stackstorm.com/development/pack_testing.html) (not required but really recommended)

### CODE REVIEW

File: packs/jira/aliases/create_issue.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}} assignee {{assignee}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}} assignee {{assignee}} labels {{labels}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}} assignee {{assignee}} labels {{labels}} components {{components}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}} assignee {{assignee}} labels {{labels}} components {{components}} priority {{priority}}"
+ack:
+    format: "Got it! Creating that issue for you... :book:."
+result:
+    format: |
+        You are now an obedient follower of Agile/Scrum BS.
```
Comment (tonybaloney) [inline packs/jira/aliases/create_issue.yaml]: hmmm

Comment (armab) [inline packs/jira/aliases/create_issue.yaml]: lol :smiley:

File: packs/jira/aliases/create_issue.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+name: "create_jira_issue"
+action_ref: "jira.create_issue"
+description: "Create a JIRA issue."
+formats:
+    - "create issue with summary {{summary}} description {{description}} of type {{type}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}} assignee {{assignee}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}} assignee {{assignee}} labels {{labels}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}} assignee {{assignee}} labels {{labels}} components {{components}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}} assignee {{assignee}} labels {{labels}} components {{components}} priority {{priority}}"
```
Comment (armab) [inline packs/jira/aliases/create_issue.yaml]: That waterfall of formats looks pretty crazy :cactus: We're agile now!

Doesn't make it somehow easier, comparing to using Web.
Probably for real life usage 1 shortest format is enough.

Comment (emedvedev) [inline packs/jira/aliases/create_issue.yaml]: Yeah. Definitely not the way to go, especially considering you can have arbitrary key=value pairs at the end, so this waterfall is redundant.

@lakshmi-kannan you're probably new to StackStorm, but you should try to read and understand the documentation before contributing ❤️

**Pull request closed without merging**